sdk/identity/azure-identity/azure/identity/aio/_credentials/azure_powershell.py

```diff
@@ -29,9 +29,9 @@ class AzurePowerShellCredential(AsyncContextManager):
 
     This requires previously logging in to Azure via "Connect-AzAccount", and will use the currently logged in identity.
 
-    :keyword List[str] additionally_allowed_tenants: Optional additional tenant ids for which the credential
-        may acquire tokens. Add the wildcard value "*" to allow the credential to acquire tokens for
-        any tenant the application is installed.
+    :keyword List[str] additionally_allowed_tenants: Specifies tenants in addition to the specified for which the
+        credential may acquire tokens. Add the wildcard value "*" to allow the credential to acquire tokens for
+        any tenant the logged in account can access.
```

Comment:
Same, Should we hold off on adding this parameter completely until we add the tenant_id parameter to the credential?